documentation/topics/security/policies.md

```diff
@@ -54,6 +54,37 @@ There are four check types, all of which do what they sound like they do:
 
 If a single check does not explicitly authorize or forbid the whole policy, then the flow moves to the next check. For example, if an `authorize_if` check does NOT return true, this _does not mean the whole policy is forbidden_ - it means that further checking is required.
 
+### Alternative Ways To Write a Policy
+
+#### Policy with keyword list
+
+```elixir
+policies do
+  policy always(), authorize_if: always()
+end
+```
+
+#### Policy with `condition` inside `do` block
+
+```elixir
+policies do
+  policy do
+    condition(always())
```

Comment:
`mix spark.formatter` does not add `condition` to `.formatter.exs`.

Reply:
I've fixed `spark.formatter` to include `condition`.